README.md

```diff
@@ -35,12 +35,14 @@ The output of libvips' [build-win64-mxe](https://github.com/libvips/build-win64-
 
 ### macOS
 
-See [package-libvips-darwin](https://github.com/lovell/package-libvips-darwin).
```

Comment:
We may have to add a depreciation note to the [`package-libvips-darwin`](https://github.com/lovell/package-libvips-darwin) repository after this.